src/pages/EnablePayments/VerifyIdentity/VerifyIdentity.tsx

```diff
@@ -0,0 +1,138 @@
+            />
+            <View style={[styles.ph5, styles.mt3, {height: CONST.BANK_ACCOUNT.STEPS_HEADER_HEIGHT}]}>
+                <InteractiveStepSubHeader
+                    startStepIndex={2}
+                    stepNames={CONST.WALLET.STEP_NAMES}
+                />
+            </View>
+            <FullPageOfflineBlockingView>
+                <ScrollView contentContainerStyle={styles.flex1}>
+                    {walletOnfidoData?.hasAcceptedPrivacyPolicy ? (
+                        <Onfido
+                            sdkToken={walletOnfidoData?.sdkToken ?? ''}
+                            onUserExit={goBack}
+                            onError={handleOnfidoError}
+                            onSuccess={handleOnfidoSuccess}
+                        />
+                    ) : (
+                        <>
+                            <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter, styles.m5, styles.ph5]}>
+                                <Icon
+                                    src={Illustrations.ToddBehindCloud}
+                                    fill="black"
+                                    width={100}
+                                    height={100}
+                                />
+                                <Text style={[styles.textHeadline, styles.mb2]}>{translate('onfidoStep.letsVerifyIdentity')}</Text>
+                                <Text style={[styles.textAlignCenter, styles.textSupporting]}>{translate('onfidoStep.butFirst')}</Text>
+                            </View>
+                            <FixedFooter>
+                                <FormAlertWithSubmitButton
+                                    isAlertVisible={Boolean(onfidoError)}
+                                    onSubmit={openOnfidoFlow}
+                                    message={onfidoError}
+                                    isLoading={walletOnfidoData?.isLoading}
+                                    buttonText={onfidoError ? translate('onfidoStep.tryAgain') : translate('common.continue')}
+                                    containerStyles={[styles.mh0, styles.mv0, styles.mb0]}
+                                />
+                            </FixedFooter>
+                        </>
+                    )}
+                </ScrollView>
+            </FullPageOfflineBlockingView>
+        </ScreenWrapper>
+    );
+}
+
+VerifyIdentity.displayName = 'VerifyIdentity';
+
+export default withOnyx<VerifyIdentityProps, VerifyIdentityOnyxProps>({
+    // @ts-expect-error: ONYXKEYS.PERSONAL_BANK_ACCOUNT is conflicting with ONYXKEYS.FORMS.PERSONAL_BANK_ACCOUNT_FORM
+    personalBankAccount: {
+        key: ONYXKEYS.PERSONAL_BANK_ACCOUNT,
+    },
+    walletOnfidoData: {
+        key: ONYXKEYS.WALLET_ONFIDO,
+
+        // Let's get a new onfido token each time the user hits this flow (as it should only be once)
+        initWithStoredValues: false,
+    },
+})(VerifyIdentity);
```

Comment:
Same here, could we refactor this to use `useOnyx`?

Reply:
@koko57 seems this is missed, let's use `useOnyx` here as well.